tutorials/marble/jupyterlab-howto.md

```diff
@@ -0,0 +1,96 @@
+# How to Use
+
+## Jupyter Images
+An image is a portable configuration of a computing instance; a snapshot of a computing environment at a point in time.
+
+If the server you are connected to has Jupyter installed you will see a dropdown list of Jupyter images after logging 
```

Comment:
Can we reword this... just because a server has jupyter installed doesn't necessarily mean that their landing page after login will take the user to the jupyterlab server launch screen. What about:

> If the server that you are connected to has Jupyter installed, you will be able to select one of the images after logging in from the main Jupyter page.

Reply:
That sounds better.